src/mumble/PulseAudio.cpp

```diff
@@ -12,6 +12,12 @@
 // We define a global macro called 'g'. This can lead to issues when included code uses 'g' as a type or parameter name (like protobuf 3.7 does). As such, for now, we have to make this our last include.
 #include "Global.h"
 
+#ifdef Q_CC_GNU
+# define RESOLVE(var) {var = reinterpret_cast<__typeof__(var)>(lib.resolve(#var)); if (!var) return; }
+#else
+# define RESOLVE(var) { *reinterpret_cast<void **>(&var) = static_cast<void *>(lib.resolve(#var)); if (!var) return; }
```

Comment:
This looks really strange to me. Isn't there a better alternative than casting everything to a `void *`?

Reply:
It's the same method we use for the codecs:
https://github.com/mumble-voip/mumble/blob/3d7f8cc4e5f502652f6660822eb791cec79a61f7/src/mumble/OpusCodec.cpp#L12-L24
https://github.com/mumble-voip/mumble/blob/3d7f8cc4e5f502652f6660822eb791cec79a61f7/src/mumble/CELTCodec.cpp#L12-L24

We could move the macro to `src/Utils.h` and then investigate whether it can be improved.